### PR TITLE
Studio Tools, DCC Scene Exporter Tooling Updates

### PR DESCRIPTION
**Pull request by HogJonny-AMZN**
## What does this PR do?
This PR is pulling severl things across the line to wrap up an Epic about SceneExport tools for Maya and Blender.  There are some updates to the tools, including some bug fixes, and updated to readme documentation for the tools.

1. Several readme's were updated 
2. Blender changes:
3. Blender Scene Exporter can be Enabled Without Manual Install
4. DCCsi has Blender\Scripts\addons, ther SceneExporter was moved down a level into here
5. config.py now sets ENVAR for BLENDER_USER_SCRIPTS
6. User can use DCCsi\Tools\DCC\Blender\start.py, and the addon is now available to directly active in the prefs (User no longer need to manually install.)
7. the readme for the Blender Addon is being updated
8. fixes for: https://github.com/o3de/o3de/issues/12965
9. add python.sh for powershell
10. updated readme, changed `python `to `.\python` to be more clear to shell users
11. https://github.com/o3de/o3de/issues/12966
12. added 3.10.x/site-packages/stub to the DCCsi 3rdParty (will avoid foundation throwing and error because the dir doesn't exist to install pkgs into)
13. added code to foundation to .touch() that path (and make sure it exists)
14. https://github.com/o3de/o3de/issues/12968
15. added "" around paths in readme files, in case user copies and pastes
16. added logging prints for user, with instructions to wrap exe path in quotes (in this argparser, I only get back what argparses passes me, and the space in 'program files' causes problems and I only get a partial path.  Might want to switch to click or another cli and see if we can improve this in the future.
17. https://github.com/o3de/o3de/issues/12971
18. dccsi\__init__.py is more forgiving if PATH_O3DE_BIN isn't set
19. This can happen if you are a developer, in a repo clone or branch that is non-standard (different from stand path patterns), or a custom build configuration for cmake (i.e., we might know ho tow to get to engine, not where to look for binaries.)  Made this more forgiving, because for instance Maya doesn't need that access to boot, however some other tool might need something the engine built (let that tool fail further downstream.)

_Please add links to any issues, RFCs or other items that are relevant to this PR._
1. I removed my .settings.local.json
2. I used: .\python .\Tools\DCC\Maya\start.py
3. allowed Maya to boot even though that ENVAR location fails to get set to a valid folder
4. I used: .\python .\Tools\DCC\Blender\start.py
5. allows Blender to boots (also without that engine path ENVAR set)
6. verified that User can enable the o3de scene exporter from the preferences (and not manual installation required.)

### CODE REVIEW

**Comment by gadams3** [comment]
While the description is helpful, it would be beneficial to add labels and make the PR title descriptive enough so that it will be more discoverable by those who don't subscribe to the notification emails or just browse the pull request dashboard.

**Review by gadams3 — APPROVED** [review]
This looks great. Would love to see you screenshots or videos going from blender to the engine.

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/Scripts/addons/SceneExporter/README.md [hunk 1]
```diff
@@ -0,0 +1,183 @@
+"""
```
[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/Scripts/addons/SceneExporter/README.md:1]: that is the long term plan, this is the short term plan

**Comment by HogJonny-AMZN** [comment]
> While the description is helpful, it would be beneficial to add labels and make the PR title descriptive enough so that it will be more discoverable by those who don't subscribe to the notification emails or just browse the pull request dashboard.

ok, I updated the PR description to the core for this PR, the other GHI's linked here were found while testing with Ben, he is updating a Maya readme and we are including it in this PR so it can all go through AR together.

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/Scripts/addons/SceneExporter/README.md [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+
+
+
+
+
+
+
+
+
+
```
[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/Scripts/addons/SceneExporter/README.md:183]: done

[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/Scripts/addons/SceneExporter/README.md:183]: removed

File: Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/start.py [hunk 3]
```diff
@@ -81,12 +81,39 @@
...
+from DccScriptingInterface.Tools.DCC.Blender import SLUG_BLENDER_EXE
+from DccScriptingInterface.Tools.DCC.Blender import PATH_DCCSI_BLENDER_LOCATION
+from DccScriptingInterface.Tools.DCC.Blender import PATH_DCCSI_BLENDER_EXE
 
-# may re-enable later
-# # retreive the blender_config class object and it's settings
-# from DccScriptingInterface.Tools.DCC.Blender.config import blender_config
-# blender_config.settings.setenv() # ensure env is set
+_BLENDER_EXE = Path(PATH_DCCSI_BLENDER_EXE).resolve(strict=True)
+#_BLENDER_EXE = Path(blender_config.settings.PATH_DCCSI_BLENDER_EXE)
```
[the remark this answers is not in the document]
Comment (HogJonny-AMZN) [reply Gems/AtomLyIntegration/TechnicalArt/DccScriptingInterface/Tools/DCC/Blender/start.py]: I cleaned up this file

**Pull request merged**